behat.yml

```diff
@@ -5,7 +5,7 @@ default:
     cache: ~
     filters:
       # Allow skipping tests by tagging them with "@skipped"
-      tags: '~@skipped'
+      tags: '~@d9&&~@d10&&~@subtheme&&~@skipped'
```

Comment:
Could you update the comment on these tags to be clear of the logic. And below please

Reply:
updated tag names to be more explicit (that it is D9 or D10 ONLY tests) and updated comments